### PR TITLE
restructured content organisation

### PR DESCRIPTION
**Pull request by MathieuMonneret**
i reused all the content that already existed but diplayed it in a more understandable way, got rid of the menu on the right, added subsections in left menu.

### CODE REVIEW

**Review by htdvisser — COMMENTED** [review]
Please make sure that redirects are created for any moved pages, otherwise existing links to our documentation will break. See also https://github.com/TheThingsNetwork/docs/blob/master/_content/network/cli/index.md#L4-L7

File: _content/lorawan/academic.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 ---
 title: Academic Research
+section: Additional Informations
```
Comment (johanstokking) [inline _content/lorawan/academic.md]: Singular

**Review by johanstokking — COMMENTED** [review]
I'm quite sure Jekyll supports alternative locations; we used that in the past. This allows for redirects generated automatically.

File: Gemfile.lock [hunk 2]
```diff
@@ -96,6 +99,7 @@ GEM
 
 PLATFORMS
   ruby
+  x64-mingw32
```
Comment (johanstokking) [inline Gemfile.lock]: What are the diffs in this file for?

File: _content/lorawan/adaptive-data-rate.md [hunk 3]
```diff
@@ -1,8 +1,12 @@
 ---
 title: Adaptive Data Rate
+section: Specifications
+zindex: 7
+redirect_from:
+ - current/adr/
 ---
 
-# LoRaWAN Adaptive Data Rate
+# Adaptive Data Rate
```
Comment (johanstokking) [inline _content/lorawan/adaptive-data-rate.md]: Restore for SEO

File: _content/lorawan/frequency-band-overview.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
+---
+title: Frequency Bands
```
Comment (johanstokking) [inline _content/lorawan/frequency-band-overview.md]: Regional Parameters, also filename

File: _content/lorawan/frequency-band-overview.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+section: Fundamental
+zindex: 3
+redirect_from:
+ - current/#frequency-bands/
+---
+
+# Frequency Bands
```
Comment (johanstokking) [inline _content/lorawan/frequency-band-overview.md]: Regional Parameters

File: _content/lorawan/security.md [hunk 6]
```diff
@@ -1,18 +1,32 @@
 ---
 title: Security
+section: Specifications
+zindex: 2
+redirect_from:
+ - current//#security/
```
Comment (johanstokking) [inline _content/lorawan/security.md]: Check this

**Review by johanstokking — CHANGES_REQUESTED** [review]

File: Gemfile.lock (hunk 2 above)
Comment (MathieuMonneret) [reply Gemfile.lock]: It was made automatically by Jekyll with my version of ruby I can delete it i think.

File: _content/lorawan/security.md (hunk 6 above)
Comment (MathieuMonneret) [reply _content/lorawan/security.md]: actually I think that the `redirect_from` are not necessary as we mentioned before.

Comment (johanstokking) [reply _content/lorawan/security.md]: Mentioned before where?

Comment (MathieuMonneret) [reply _content/lorawan/security.md]: https://github.com/TheThingsNetwork/docs/pull/319#pullrequestreview-291694400

Comment (johanstokking) [reply _content/lorawan/security.md]: That is exactly what I meant with using `redirect_from`. Those are creating the redirects for SEO.

Comment (johanstokking) [reply _content/lorawan/security.md]: I'm triggered by the double `/` in this URL; that doesn't seem a valid origin

Comment (MathieuMonneret) [reply _content/lorawan/security.md]: check latest commit.

File: _content/lorawan/security.md [hunk 7]
```diff
@@ -3,7 +3,7 @@ title: Security
 section: Specifications
 zindex: 2
 redirect_from:
- - current//#security/
+ - ./index/#security/
  - current/security/
```
Comment (johanstokking) [inline _content/lorawan/security.md]: I still don't get which redirects these are intended to be. Can you specify the existing sources that are being redirected here?

Comment (MathieuMonneret) [reply _content/lorawan/security.md]: I am not sure about the `current` maybe I should replace it with `lorawan`.
line 6 is to redirect from the sub-part `security` in `index.md`.

Comment (johanstokking) [reply _content/lorawan/security.md]: https://github.com/jekyll/jekyll-redirect-from#usage

File: _content/lorawan/duty-cycle.md [hunk 8]
```diff
@@ -3,7 +3,6 @@ title: Duty Cycle
 section: Specifications
 zindex: 6
 redirect_from:
- - /lorawan/duty-cycle
```
Comment (MathieuMonneret) [inline _content/lorawan/duty-cycle.md]: actually url didn't change so unnecessary.

File: _content/lorawan/security.md [hunk 9]
```diff
@@ -4,7 +4,6 @@ section: Specifications
 zindex: 2
 redirect_from:
  - /lorawan/#security
- - /lorawan/security
```
Comment (MathieuMonneret) [inline _content/lorawan/security.md]: same here.

File: _content/lorawan/LoRaWAN-architecture.md [hunk 10]
```diff
@@ -3,7 +3,7 @@ title: LoRaWAN Architecture
 section: Fundamental
 zindex: 4
 redirect_from:
- - current/#terminology/
+ - /lorawan/#terminology
```
Comment (johanstokking) [inline _content/lorawan/LoRaWAN-architecture.md]: I don't think we can redirect from anchors

File: _content/lorawan/academic.md [hunk 11]
```diff
@@ -3,7 +3,7 @@ title: Academic Research
 section: Additional Information
 zindex: 10
 redirect_from:
- - current/academic/
+ - /lorawan/academic
```
Comment (johanstokking) [inline _content/lorawan/academic.md]: What is the redirect here really? Isn't this file the same location?

File: _content/lorawan/adaptive-data-rate.md [hunk 12]
```diff
@@ -3,7 +3,7 @@ title: Adaptive Data Rate
 section: Specifications
 zindex: 7
 redirect_from:
- - current/adr/
+ - /lorawan/adr
```
Comment (johanstokking) [inline _content/lorawan/adaptive-data-rate.md:6]: Did you test that this creates a redirect from `/lorawan/adr.html`? Otherwise it doesn't work

**Review by johanstokking — COMMENTED** [review]
- Use lowercase names, i.e. `architecture.md` instead of `LoRaWAN-...`

File: _content/lorawan/LoRaWAN-architecture.md [hunk 13]
```diff
@@ -3,7 +3,6 @@ title: LoRaWAN Architecture
 section: Fundamental
 zindex: 4
 redirect_from:
- - /lorawan/#terminology
```
Comment (johanstokking) [inline _content/lorawan/LoRaWAN-architecture.md]: Remove unnecessary `redirect_from` if empty

**Review by johanstokking — APPROVED** [review]

**Pull request merged**